cookbooks/aws-parallelcluster-platform/test/controls/dcv_spec.rb

```diff
@@ -54,7 +54,7 @@
 
 control 'tag:install_dcv_installed' do
   title 'Check dcv is installed'
-  only_if { !os_properties.redhat_on_docker? && !(os_properties.ubuntu? && os_properties.arm?) }
+  only_if { !os_properties.redhat_on_docker? && !(os_properties.ubuntu? && os_properties.arm?) && !os_properties.alinux2023? }
```

Comment:
what about having a 'is_dcv_supported'? to prevent the redefinition of the conditions and improve readability?

Reply:
Since this is in test code, can we address this later?

Reply:
yes